src/hyp3_isce2/insar_tops_burst.py

```diff
@@ -363,6 +367,37 @@ def main():
     product_dir.mkdir(parents=True, exist_ok=True)
 
     translate_outputs(isce_output_dir, product_name)
+
+    payload = {}
+    payload['product_dir'] = Path(product_name)
+    payload['reference_granule_name'] = args.reference_scene
+    payload['secondary_granule_name'] = args.secondary_scene
+    payload['swath_number'] = args.swath_number
+    payload['polarization'] = args.polarization
+    payload['reference_burst_number'] = args.reference_burst_number
+    payload['secondary_burst_number'] = args.secondary_burst_number
+    payload['processing_date'] = datetime.now(timezone.utc)
+    payload['range_looks'] = args.range_looks
+    payload['azimuth_looks'] = args.azimuth_looks
+    payload['dem_name'] = 'GLO-30'
```

Comment:
Because we just use the one DEM, that changes the way we deal with DEM-related information. Is there a place that we define the pixel spacing of the GLO-30 DEM, or should we just hard-code that value on line 199 of the metadata template?

Reply:
previous I use marco to get the DEM pixel spacing of GLO-30 DEM, since we do not want to use the macro, we need hard code, or we can leave it as jinjn replacement, and use the payload to replace it. Either way is OK.

Reply:
If it's okay to just add a payload entry, that's probably the most transparent. It probably doesn't make much difference, as if we ever change the DEM, we'll have to completely overhaul the DEM description sections, but it somehow seems more robust to me.

Reply:
Resolved with https://github.com/ASFHyP3/hyp3-isce2/pull/76/commits/d397179508442e956a2de8ee4ce20a64c3f85e6f